math/wide_integer/uintwide_t.h

```diff
@@ -1070,11 +1080,11 @@
       using my_own::frexp;
 
       // Get the fraction and base-2 exponent.
-      native_float_type man = (native_float_type) frexp(f, &my_exponent_part);
+      auto man = static_cast<native_float_type>(frexp(f, &my_exponent_part));
 
       unsigned n2 = 0U;
 
-      for(std::uint_fast16_t i = static_cast<std::uint_fast16_t>(0U); i < static_cast<std::uint_fast16_t>(std::numeric_limits<native_float_type>::digits); ++i)
+      for(auto i = static_cast<std::uint_fast16_t>(0U); i < static_cast<std::uint_fast16_t>(std::numeric_limits<native_float_type>::digits); ++i)
```

Comment:
What is the benefit of `std::uint_fast16_t` over `int`? It's very rare that anything can outperform `int` and (shockingly) `sizeof(std::uint_fast16_t)` appears to be 8 bytes!